orchestra/static/orchestra/todos/todolist-template.service.es6.js

```diff
@@ -0,0 +1,18 @@
+export default function todoListTemplateApi ($http) {
+  const apiBase = '/orchestra/todos/'
+
+  const list = () => {
+    return `${apiBase}todolist_templates/`
+  }
+
+  const addTodoListTemplate = () => {
```

Comment:
high-level throughout: should we be "creating/updating todo list from template" or "adding todo list template?" My vote is the first one, meaning in this example it's `createTodoListFromTemplate` or `updateTodoListFromTemplate`

Reply:
I like `updateTodoListFromTemplate`! It clear explains that we updating TodoList. But in such a case I would prefer to change the API URL to `update_todos_from_todolist_template`.  What do you think?

I dont like `createTodoListFromTemplate` because we are actually appending the todos from the template.